src/node.cc

```diff
@@ -4114,10 +4114,11 @@ inline void PlatformInit() {
       continue;
     // Anything but EBADF means something is seriously wrong.  We don't
     // have to special-case EINTR, fstat() is not interruptible.
-    // if (errno != EBADF)
-    //   ABORT();
-    if (fd != open("/dev/null", O_RDWR))
+    if (errno != EBADF)
       ABORT();
+    open("/dev/null", O_RDWR);
```

Comment:
I think we should make sure that this open return current loop fd (STDIN_FILENO, STDOUT_FILENO, STDERR_FILENO).